WordPress/src/main/java/org/wordpress/android/ui/notifications/adapters/NotesAdapter.kt

```diff
@@ -301,156 +272,110 @@ class NotesAdapter(
     fun reloadLocalNotes() {
         cancelReloadLocalNotes()
         reloadLocalNotesJob = coroutineScope.launch {
-            val notes = NotificationsTable.getLatestNotes()
-            addAll(notes)
+            addAll(NotificationsTable.getLatestNotes())
         }
     }
 
     /**
      * Update the note in the adapter and notify the change
      */
     fun updateNote(note: Note) {
-        val notePosition = filteredNotes.indexOfFirst { it.id == note.id }
-        if (notePosition != -1) {
+        filteredNotes.indexOrNull { it.id == note.id }?.let { notePosition ->
             filteredNotes[notePosition] = note
             notifyItemChanged(notePosition)
         }
     }
 
-    inner class NoteViewHolder(view: View) : RecyclerView.ViewHolder(view) {
```

Comment:
Extract NoteViewHolder from NoteAdapter and use ViewBinding

Reply:
Thank you 🙇 
This makes sense and makes the `NotesAdapter` a lot more readable 🏅